test/hotspot/jtreg/runtime/cds/appcds/jcmd/JCmdTestFileSafety.java

```diff
@@ -64,6 +64,13 @@ static void checkContainAbsoluteLogPath(OutputAnalyzer output) throws Exception
         }
     }
 
+    private static void removeFile(String fileName) throws Exception {
+        File file = new File(fileName);
+        if (file.exists()) {
+            file.delete();
+        }
+    }
```

Comment:
This method should eventually be moved to JCmdTestDumpBase.java because the other test
(JCmdTestFileSafety.java) could make use of it.